docs/pbs/index.md

```diff
@@ -312,10 +312,22 @@ The server-specific queue names will be understood by both PBS servers,
 so if you will want to submit the same script at times from Casper,
 *always append the server name to your queue*.
 
-The `qinteractive` and `execcasper` scripts, which start interactive
-jobs on Casper, will adjust the queue name for
-you to include the server, so *you do not need to append the server name
-manually for interactive jobs*.
+### Interactive Jobs
+
+Users can start an interactive job on `casper` or `Derecho` using the `qsub -I` command. The `-I` flag is used to request an interactive job. The following example shows how to start an interactive job on `casper`:
+
+```bash
+qsub -I -l select=1:ncpus=1:mem=20GB -q casper@casper-pbs -l walltime=06:00:00 -A <project_code>
+```
+
+Additionally, the `qinteractive` command can be used to start an interactive job on `derecho` or `casper`. For `derecho` this will start an interactive job on the `develop` queue, while for `casper` it will submit the job to the `main` queue. The following example shows how to start an interactive job on `derecho` or `casper`:
+
+```bash
+qinteractive -A <project_code>
+```
+
+Additionally `execasper` can be used to start an interactive job on `casper`. If you are on a `derecho` login node, using `execasper` will start an interactive job on `casper`. 
```

Comment:
These scripts are more designed to focus specifically on one system or the other. I'd suggest this:

----

We provide convenience scripts for starting interactive jobs on both systems. If you wish to start an interactive session on Derecho, you can use the `qinteractive` command. Meanwhile, the `execcasper` command will start an interactive session on Casper. For example:

```bash
execcasper -A <project_code>
```
By default, each command will assign you the following resources:
* `qinteractive` - 32 cores with 55 GB of memory on a single node in the `develop` queue
* `execcasper` - 1 core with 10 GB of memory in the `casper` queue

Job settings can be customized using typical qsub arguments. The scripts also provide additional arguments like `--mem` to simplify submission. Run either command with `--help` for more information. Furthermore, both scripts will detect whether you have X11 support in your terminal and automatically configure your job environment accordingly.

Reply:
@vanderwb I think that might not be 100% correct. `qinteractive` on casper will submit this job on Casper (and not Derecho): 

```
qsub -I -l select=1:ncpus=1:mem=10GB -q casper@casper-pbs -l walltime=06:00:00
```

But `execcasper` on both machines submit jobs to Casper.

Reply:
Roger - that's a bug with qinteractive then. I'll get that fixed.